### ISSUE TITLE
Feature Request - Data Source for MVE Image IDs

### ISSUE THREAD

**Issue by BrokeAssChewbacca** [megaport/terraform-provider-megaport#47]
# Problem

- When utilizing the `megaport_mve` resource you are required to pass in the `image_id` that you wish to use.
- Currently you have to utilize the Megaport API in order to retrieve these values (using the locations API is where I've found them) and build custom logic in Terraform to get these values

# Proposed Solution

- Create a new Terraform Data source that will allow users to pass in attributes to retrieve the Image ID, example:

```hcl
data "megaport_mve_image" "palo_alto_vm-series" {
  vendor  = "PALO_ALTO"
  product = "VM-Series"
  version = "10.2.0"
}

# In this case: data.megaport_mve_image.palo_alto_vm-series.id would return 32
```

**Comment by mega-alex** [comment]
Hi there, we're looking into this and it is on our roadmap with the new provider.

**Comment by mega-alex** [comment]
This has been addressed in the `v1.0.1-rc1` version of the provider.

**Comment by Kostoprav19** [comment]
@mega-alex please give an example how to use mve data source.

For example  how to get image_id=58, knowing product_size, vendor and possibly version of fortios
```
# Megaport MVE - Fortigate router
resource "megaport_mve" "fortigate" {
  product_name         = "TEST-MVE"
  location_id          = data.megaport_location.this.id
  contract_term_months = "1"

  vendor_config = {
    vendor         = "fortinet"
    image_id       = 58
    product_size   = "MEDIUM"
    ssh_public_key = XXX
    license_data   = YYY
  }

}
```
